astro/src/content/docs/customize/look-and-feel/upgrade.mdx

```diff
@@ -20,6 +20,16 @@ New versions of FusionAuth sometimes include new or updated theme templates. If
 
 This page contains notes on changes in recent versions of FusionAuth that require changes to a customized theme.
 
+## Version 1.53.3
+
+Version `1.53.3` includes a change to persist the user's SSO decision through an external identity provider workflow. If the Google IdP's `loginMethod` is configured as `UsePopup` or `UseVendorJavaScript`, existing custom advanced themes require an update to the `helpers` template to incorporate the fix for the Google IdP. You can update the template via the API or by modifying the *Helpers* template in the admin UI.
+
+To allow the user's SSO decision to be persisted through a Google IdP login in an existing custom advanced theme, remove the `data-login_uri` attribute from the `div` with Id `g_id_onload` in the `googleButton` macro and add the following attribute in its place:
```

Comment:
Is there a link to the relevant Google documentation that we can add here? I did a very brief search and it was not obvious where this came from, but it could provide some useful context (if it exists)

Reply:
It's [here](https://developers.google.com/identity/gsi/web/reference/html-reference#data-callback), but I don't feel like it's that useful.